addon/components/o-s-s/phone-number-input.ts

```diff
@@ -0,0 +1,81 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { countries, CountryData } from '@upfluence/oss-components/utils/country-codes';
+
+interface OSSPhoneNumberInputArgs {
+  prefix: string;
+  number: string;
+  onChange: (prefix: string, number: string) => void;
```

Comment:
```suggestion
  onChange(prefix: string, number: string): void;
```
We don't use this syntax now?

Reply:
We sure do !